README.md

```diff
@@ -3,7 +3,7 @@ Kashoo IAP Tester
 =================
 
 Created by Ben Kennedy on 2013-03-22.
-Copyright (c) 2013 Kashoo Inc. All rights reserved.
+Copyright (c) 2016 Kashoo Inc. All rights reserved.
```

Comment:
I don't think it really matters. In fact, as I understand it, we could leave the copyright notice out completely and it would make no difference, legally speaking.